R/pkg/R/DataFrame.R

```diff
@@ -177,8 +177,8 @@ setMethod("isLocal",
 #' @param x A SparkDataFrame
 #' @param numRows The number of rows to print. Defaults to 20.
 #' @param truncate Whether truncate long strings. If true, strings more than 20 characters will be
-#' truncated and all cells will be aligned right
-#'
+#'    truncated. However, if set greater than zero, truncates strings longer than `truncate`
```

Comment:
Code change looks fine. Can we add a R unit test as well ? We could just add another test similar to https://github.com/ScrapCodes/spark/blob/daab62a4f3fa8515fbe3d787e489abbc69370628/R/pkg/inst/tests/testthat/test_sparkSQL.R#L1578 but with say `truncate=3`